rules_csharp_gapic/BUILD.bazel

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load(":csharp_compiler.bzl", "csharp_binary")
+
+csharp_binary(
+    name = "csharp_gapic_generator_binary",
+    visibility = ["//visibility:public"],
+    restore_packages = "@gapic_generator_restore//:packages",
+    restore_obj = "@gapic_generator_restore//:obj",
+    src_base = "@gapic_generator_src//:gen_dest/Google.Api.Generator",
+    csproj_name = "Google.Api.Generator.csproj",
+    runtime = "linux-x64",
```

Comment:
The binary should be built for the host platform automatically (on which the bazel build command is executed). I see that this is an initial implementation of the rules and not everything is handled yet, but the platform should not be an argument of a build rule itself (it undermines many fundamental bazel's principles). Why is this needed?